bridge/bridge.go

```diff
@@ -19,10 +24,18 @@ func (b PdkBridge) Ask(method string, args ...interface{}) (string, error) {
 	} else {
 		call := method + ":" + string(argsJson)
 		b.ch <- call
-		if reply := <-b.ch; reply == "null" {
-			return "", errors.New("null response")
+		reply := <-b.ch
+		res := Response{}
+
+		if err := json.Unmarshal([]byte(reply), &res); err != nil {
```

Comment:
now that it's always JSON on both call and response, it's getting more important to test msgpack...

Reply:
agreed! we will be looking at the serialization format next